src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataUpdater.java

```diff
@@ -89,13 +93,19 @@ private void createNewBlogger(BloggerEntry bloggerEntry,
         updateSummary.recordCreated();
     }
 
-    protected boolean isEqual(Blog blog, BloggerEntry bloggerEntry) {
-        return Objects.equals(blog.getAuthor(), bloggerEntry.getName())
-            && Objects.equals(blog.getJsonId(), bloggerEntry.getJsonId())
-            && StringUtils.equalsIgnoreCase(blog.getRss(), bloggerEntry.getRss())
-            && StringUtils.equalsIgnoreCase(blog.getUrl(), bloggerEntry.getUrl())
-            && Objects.equals(blog.getBlogType(), bloggerEntry.getBlogType())
-            && Objects.equals(blog.getTwitter(), bloggerEntry.getTwitter());
+    protected boolean somethingChangedInBloggerData(Blog blog, BloggerEntry bloggerEntry) {
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2\.